src/ivory-px4-hw/calibration.conf

```diff
@@ -1,8 +1,32 @@
-[calibration.accelerometer]
-x_offset=0
-y_offset=0
-z_offset=0
-[calibration.magnetometer]
-x_offset=0
-y_offset=0
-z_offset=0
+[calibration.mpu6000.accelerometer]
+x_offset=0.0
+y_offset=0.0
+z_offset=0.0
+x_scale=1.0
+y_scale=1.0
+z_scale=1.0
+[calibration.mpu6000.gyroscope]
```

Comment:
We still need the scale, so I think this should stay. Otherwise we'd need a special case to handle calibrations that are scale-only